gammapy/stats/feldman_cousins.py

```diff
@@ -0,0 +1,444 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Provide a Feldman Cousins algorithm."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from astropy.extern.six.moves import range
+from astropy.extern.six import iteritems
+
+import logging
+log = logging.getLogger(__name__)
+
+__all__ = ['fc_find_acceptance_region_gauss',
```

Comment:
Feel free to ignore me here, but just as a question / suggestion: is it possible to make the function names shorter / simpler / more uniform without being cryptic?

One possible improvement would be to always use "interval" instead of sometimes "region".
FC, at least as implemented here and commonly used, only works for one parameter, so "interval" is a bit more clear, no?

And maybe `fc_get_limits_poisson` would be a bit better than `fc_find_acceptance_interval_poisson`, because it's shorter and it's a special case of the more general `fc_get_limits` function? (I didn't look closely if this is actually the case, if this is incorrect and you think the current function names are best, just ignore me here.)